src/main/webapp/resources/js/utilities/antdesign-table-utilities.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * This file is for any dynamic settings required for an ant design table.
+ */
+
+/**
+ * Default page size for ant design tables
+ */
+export const defaultPageSize = 10;
+
+export const defaultShowPageSizeChanger = true;
+
+/**
+ * Set the common pagination options for the tables
+ */
+export function getPaginationOptions(totalEntries) {
+  if (totalEntries <= 10) {
+    return {
+      pageSizeOptions: ["10"],
```

Comment:
Why are you adding these as strings?  In the docs they are just numbers.

Reply:
Didn't test it out with just numbers. I always had it as strings in other places as well but can update it

Reply:
Updated in c96bd77d27718c9064e6535c7aac61dd11b3f80b